docker-unified/hbs/nginx-config.hbs

```diff
@@ -1,58 +1,63 @@
 server {
   listen 80 default_server;
   #listen [::]:80 default_server;
-  server_name {{ getenv "BW_DOMAIN" "localhost" }};
-{{ if eq (getenv "BW_ENABLE_SSL") "true" }}
+  server_name {{{String.Coalesce env.BW_DOMAIN "localhost"}}};
+{{#if (String.Equal env.BW_ENABLE_SSL "true")}}
 
-  return 301 https://{{ getenv "BW_DOMAIN" "localhost" }}$request_uri;
+  return 301 https://{{{String.Coalesce env.BW_DOMAIN "localhost"}}}$request_uri;
 }
 
 server {
   listen 443 ssl http2;
   #listen [::]:443 ssl http2;
-  server_name {{ getenv "BW_DOMAIN" "localhost" }};
+  server_name {{{String.Coalesce env.BW_DOMAIN "localhost"}}};
 
-  ssl_certificate /etc/bitwarden/{{ getenv "BW_SSL_CERT" "ssl.crt" }};
-  ssl_certificate_key /etc/bitwarden/{{ getenv "BW_SSL_KEY" "ssl.key" }};
+  ssl_certificate /etc/bitwarden/{{{String.Coalesce env.BW_SSL_CERT "ssl.crt"}}};
+  ssl_certificate_key /etc/bitwarden/{{{String.Coalesce env.BW_SSL_KEY "ssl.key"}}};
   ssl_session_timeout 30m;
   ssl_session_cache shared:SSL:20m;
   ssl_session_tickets off;
+{{#if (String.Equal env.BW_ENABLE_SSL_DH "true")}}
 
-  ssl_protocols {{ getenv "BW_SSL_PROTOCOLS" "TLSv1.2" }};
-  ssl_ciphers "{{ getenv "BW_SSL_CIPHERS" "ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256" }}";
+  # Diffie-Hellman parameter for DHE ciphersuites, recommended 2048 bits
+  ssl_dhparam /etc/bitwarden/{{{String.Coalesce env.BW_SSL_DH_CERT "dh.pem"}}};
+{{/if}}
```

Comment:
I also added back DHE params, which seemed to get dropped in translation from the standard deployment.